NameLayerMain/src/vg/civcraft/mc/namelayer/command/commands/DisciplineGroup.java

```diff
@@ -42,7 +42,6 @@ public boolean execute(CommandSender sender, String[] args) {
 			g.setDisciplined(true);
```

Comment:
Yeah, this doesn't update the DB at all. 

So this isn't a problem you introduced, Suirad, but it's one you can fix -- layer in similar pattern persistence of Disciplined so that:

* It is persisted to the DB at command-source instance (here)
* The mercury message is sent and actually parsed in MercuryListener to update the other shard's caches.